ee/ucp/admin/configure/collect-cluster-metrics.md

```diff
@@ -84,8 +84,12 @@ To deploy Prometheus on worker nodes in a cluster:
     $ kubectl label node 3a724a-1 ucp-metrics=
     node "test-3a724a-1" labeled
     ```
+    
+     > **SELinux Prometheus Deployment for UCP 3.1.0-3.1.2**
```

Comment:
@L-Hudson I'm torn between leaving the versioning like this or specifically call out `UCP 3.1.0, 3.1.1, and 3.1.2` given how this note will not apply to `UCP 3.1.3`.

Reply:
Fixed